SDK/MobileCenter/Microsoft.Azure.Mobile.UWP/Utils/DefaultScreenSizeProvider.cs

```diff
@@ -33,13 +33,13 @@ public DefaultScreenSizeProvider()
                 Task.Factory.StartNew(() =>
                 {
                     var displayInfo = DisplayInformation.GetForCurrentView();
-                    UpdateDisplayInformation(displayInfo, null);
+                    UpdateDisplayInformation((int)displayInfo.ScreenHeightInRawPixels, (int)displayInfo.ScreenWidthInRawPixels);
```

Comment:
It is not possible to create DisplayInformation object other than using
```DisplayInformation.GetForCurrentView()```
method, so to test size changed event i've made some refactoring.